packages/components/src/presentation-components/properties/inputs/UniquePropertyValuesSelector.tsx

```diff
@@ -160,6 +162,8 @@ function useUniquePropertyValuesRuleset(descriptorRuleset?: Ruleset, field?: Fie
             specifications: [
               {
                 specType: "SelectedNodeInstances",
+                acceptableClassNames: selectedClasses ? selectedClasses.map(({ name }) => name.split(":")[1]) : undefined,
```

Comment:
I thinks it's better to use `split(/[\.:]/)` as this might not be coming from descriptor. So just to be safe.

Reply:
changed